tests/Iconv/IconvTest.php

```diff
@@ -106,11 +107,12 @@ public function testIconvMimeDecode()
 
     /**
      * @covers Symfony\Polyfill\Iconv\Iconv::iconv_mime_decode
+     * @expectedException \PHPUnit\Framework\Error\Notice
+     * @expectedExceptionMessage Detected an illegal character in input string
      */
     public function testIconvMimeDecodeIllegal()
     {
         iconv_mime_decode('Legal encoded-word: =?utf-8?Q?*?= .');
-        $this->setExpectedException('PHPUnit_Framework_Error_Notice', 'Detected an illegal character in input string');
```

Comment:
this test should be split in 2, so that the exception is allowed only in the second call, not the first one.

Reply:
or it should use a method_exists check to keep configuring it inline